frontend/src/views/HomeView.vue

```diff
@@ -150,6 +158,17 @@ onMounted(async () => {
       You currently have no past polls
     </p>
   </section>
+  <section class="pt-5" v-else>
+    <h2 class="capitalize text-white text-2xl font-bold mb-4">Invalid network detected</h2>
+    <p class="text-white text-base mb-20">
+      In order to continue to use the app, please switch to the correct chain, with clicking the
+      bellow "Add network" button
+    </p>
+
+    <div class="flex justify-center">
+      <AppButton variant="secondary" @click="addNetwork">Add network</AppButton>
```

Comment:
Is there a reason why not just "Switch Network"?

```suggestion
      <AppButton variant="secondary" @click="switchNetwork">Switch network</AppButton>
```

Reply:
Right, nice catch 👍